src/System.Net.Requests/src/System/Net/HttpWebRequest.cs

```diff
@@ -396,6 +397,15 @@ public override Stream EndGetRequestStream(IAsyncResult asyncResult)
                     handler.Proxy = _proxy;
                 }
 
+                // Set relevant properties from ServicePointManager
+                handler.CheckCertificateRevocationList = ServicePointManager.CheckCertificateRevocationList;
+                RemoteCertificateValidationCallback rcvc = ServicePointManager.ServerCertificateValidationCallback; // TODO #11881: When HttpWebRequest.ServerCertificateValidationCallback added, check that first
```

Comment:
I neglected to address this nit; if I need to restart all of the legs for some reason, I'll address it, otherwise I'll let it go through.  We'll be removing this TODO shortly anyway.